infrastructure/cdn-in-a-box/traffic_ops/run-go.sh

```diff
@@ -74,7 +74,11 @@ touch /var/log/traffic_ops/traffic_ops.log
 # enroll in the background so traffic_ops_golang can run in foreground
 TO_USER=$TO_ADMIN_USER TO_PASSWORD=$TO_ADMIN_PASSWORD to-enroll $(hostname -s) &
 
-./bin/traffic_ops_golang -cfg $CDNCONF -dbcfg $DBCONF -riakcfg $RIAKCONF &
+if [[ "$ADD_DEMO1_SSLKEYS" = false ]]; then
+    ./bin/traffic_ops_golang -cfg $CDNCONF -dbcfg $DBCONF & 
+else 
+    ./bin/traffic_ops_golang -cfg $CDNCONF -dbcfg $DBCONF -riakcfg $RIAKCONF &
```

Comment:
I think the point is that the variable name is `ADD_DEMO1_SSLKEYS` but it's used to remove `raikcfg`.
If I hadn't read this PR, I would have thought it was simply removing SSLKEY of this Delivery Service.
DEMO1 should correctly provide HTTP service and I can manually add SSLKEY for it as well.